lib/node_modules/@stdlib/blas/ext/base/dnannsumkbn2/src/addon.c

```diff
@@ -0,0 +1,58 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/blas/ext/base/dnannsumkbn2.h"
+#include "stdlib/napi/export.h"
+#include "stdlib/napi/argv.h"
+#include "stdlib/napi/argv_int64.h"
+#include "stdlib/napi/argv_strided_float64array.h"
+#include <node_api.h>
+#include <assert.h>
+
+/**
+* Receives JavaScript callback invocation data.
+*
+* @private
+* @param env    environment under which the function is invoked
+* @param info   callback data
+* @return       Node-API value
+*/
+static napi_value addon(napi_env env, napi_callback_info info) {
+    STDLIB_NAPI_ARGV(env, info, argv, argc, 5);
+    STDLIB_NAPI_ARGV_INT64(env, N, argv, 0);
+    STDLIB_NAPI_ARGV_INT64(env, strideX, argv, 2);
+    STDLIB_NAPI_ARGV_INT64(env, strideOut, argv, 4);
+    STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY(env, X, N, strideX, argv, 1);
+    STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY(env, Out, N, strideOut, argv, 3);
+
+    int64_t io;
+    if (strideOut < 0) {
+        io = -strideOut;
+    } else {
+        io = 0;
+    }
+
+    double *out = (double *)Out;
+    int64_t n;
+    out[io] = stdlib_strided_dnannsumkbn2(N, (double *)X, strideX, &n);
```

Comment:
```suggestion
    out[io] = stdlib_strided_dnannsumkbn2(N, X, strideX, &n);
```

Please read up on the documentation and the source code in the various macro packages. This explicit cast is not necessary here. Same thing for `Out` above.